common/const.go

```diff
@@ -0,0 +1,20 @@
+package common
```

Comment:
common is not a good name on something that can be exported.
For now, I think you should rename the folder "common" as "internal".
Once we want to export something, we should put it in an appropriate catalogue, possible insida root "pkg" directory.